modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,271 @@
+
+* With this update, {tekton-results} provides up to 100 megabytes of logging information for a pipeline or task.
+
+* With this update, any authenticated user can view the `tekton-results-api-service` route in the openshift-pipelines namespace to interact with Tekton Results using a REST API.
+
+* With this update, the {tekton-results} API includes a new endpoint for fetching summary and aggregation for a list of records.
+
+* With this update, the `GetLog` endpoint of the {tekton-results} API returns raw bytes with the `text/plain` content type.
+
+With this update, you can optionally specify a custom CA certificate in the `options.configMaps.tekton-results-api-config.data.config.DB_SSLROOTCERT` spec in the `TektonResult` CR. In this value. In this case, Tekton Results requires an SSL connection to the database server and uses this certificate for the connection.
+
+[id="breaking-changes-1-14_{context}"]
+== Breaking changes
+
+* With this update, when using the Bundles resolver, you can no longer specify the `serviceAccount` parameter. Instead, you can specify the `secret` parameter to provide the name of a secret containing authentication information for the registry. You must update any tasks or pipelines that use the `serviceAccount` parameter of the bundles resolver to use the `secret` parameter instead. The `pipeline.bindles-resolver-config.default-service-account` spec in the TektonConfig CR is no longer supported.
+
+[id="known-issues-1-14_{context}"]
+== Known issues
+
+* The `tkn pipeline logs` command does not display the logs of tasks that were defined in a pipeline with the `retries: X` parameter.
+
+[id="fixed-issues-1-14_{context}"]
+== Fixed issues
+
+* Before this update, when using GitHub Enterprise, an incoming webhook did not work. With this update, you can use incoming webhooks with GitHub Enterprise.
+
+* Before this update, if a task run or pipeline run disabled timeouts, {pipelines-shortname} would run a series of rapid reconciliations on the task run or pipeline run, degrading the performance of the controller. With this update, the controller reconciles task runs and pipeline runs with disabled timeouts normally.
+
+* Before this update, if you used a custom namespace to install Tekton Hub, the installation deleted the `openshift-pipelines` namespace, removing the {pipelines-shortname} installation. With this update, you can use a custom namespace to install Tekton Hub and your {pipelines-shortname} installation is unaffected
+
+* Before this update, when using {pac} with GitLab, if the user triggered a pipeline run by using a comment in a merge request such as `/test`, {pac} did not report the status of the pipeline run on the merge request. With this update, {pac} correctly reports the status of the pipeline run on the merge request.
+
+* Before this update, when using {pac} with GitHub Enterprise, an incoming webhook did not work. With this update, the incoming webhook works with GitHub Enterprise.
```

Comment:
this is repetition of one in line number 253?

Reply:
fixing.